widget/ui/src/containers/SwapInput/TokenSection.types.ts

```diff
@@ -3,6 +3,7 @@ export type TokenSectionProps = {
   tokenImage: string;
   tokenSymbol: string;
   chain: string;
+  mode?: 'To' | 'From';
```

Comment:
Please change it to `chianImageId?: string`

Reply:
Done